src/__experimental__/components/fieldset/fieldset.component.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { validProps } from '../../../utils/ether';
+import tagComponent from '../../../utils/helpers/tags';
+import { FieldsetStyle, LegendStyle } from './fieldset.style';
+
+class Fieldset extends React.Component {
```

Comment:
This looks like a good candidate for a purely functional component.

Reply:
I did have it functional early on, but `validProps` expects a `constructor` on the instance, so I moved to a React `Component`.

Don't know if there's a way around this?

Reply:
Just destructure the valid props yourself.

Reply:
If you wanted to you could write a version of the helper which doesn't rely on their being an instance and use that.